spec/system/admin/orders_spec.rb

```diff
@@ -592,7 +592,18 @@
           perform_enqueued_jobs(only: BulkInvoiceJob)
 
           expect(page).to have_content "Bulk Invoice created"
-          expect(page).to have_link(class: "button", text: "VIEW FILE", href: /invoices/)
+
+          within ".modal-content" do
+            expect(page).to have_link(class: "button", text: "VIEW FILE", href: /invoices/)
+
+            extract_pdf_content # extracts content do variable invoice_content
```

Comment:
It would be better to return the content from the method:
```suggestion
            invoice_content = extract_pdf_content
```

Then the test can store it however it wants and you don't have to know the name of the variable.